addons/mail/static/src/scss/composer.scss

```diff
@@ -104,6 +104,9 @@
         .o_chatter_composer_info, .o_composer_suggested_partners {
             flex: 0 0 100%;
             margin-bottom: $o-mail-chatter-gap*0.5;
+            .o_suggested_hidden {
+                display:none;
```

Comment:
whitespace between key and value, but I think you could drop this addition if you use `o_hidden` instead